src/cromshell/utilities/CromshellConfig.py

```
@@ -0,0 +1,127 @@
+            cls.is_verbose = True
+        else:
+            cls.is_verbose = False
+
+    @staticmethod
+    def __get_config_dir():
+        """Get Path To Cromshell Hidden Directory"""
+
+        config_path = str(Path.home()) + "/.cromshell"
+        Path.mkdir(Path(config_path), exist_ok=True)
+        return config_path
+
+    @staticmethod
+    def __get_submission_file(config_dir):
+        """Get File Path To Cromshell Submission File"""
+
+        submission_file = "{}/all.workflow.database.tsv".format(config_dir)
+        if not Path.exists(Path(submission_file)):
+            Path.touch(Path(submission_file))
+            submission_header = (
+                "DATE{0}CROMWELL_SERVER{0}RUN_ID{0}WDL_NAME{0}STATUS{0}ALIAS".format(
+                    "\t"
+                )
+            )
+            with Path(submission_file).open() as f:
+                f.write(submission_header)
+        return submission_file
+
+    @staticmethod
+    def __get_cromwell_server(config_dir):
+        """Get Cromshell Server URL from Configuration File"""
+
+        cromwell_server_path = "{}/cromwell_server.config".format(config_dir)
+        if Path.exists(Path(cromwell_server_path)):
+            with Path(cromwell_server_path).open() as f:
+                cromwell_server = f.readline().strip()
+            return cromwell_server
+            CromshellConfig.LOGGER.info(
+                "Setting cromwell server to cromwell url from config file."
+            )
+            CromshellConfig.LOGGER.info(cromwell_server)
+        else:
+            CromshellConfig.LOGGER.error("Could not find cromwell_server.config")
+            CromshellConfig.LOGGER.info("Creating file %s", cromwell_server_path)
+            Path.touch(Path(cromwell_server_path))
+            CromshellConfig.LOGGER.error(
+                "Please add a Cromwell server URL to file %s", cromwell_server_path
+            )
+            sys.exit()
```

Sys.exit is not ideal I think.  Maybe throw an exception instead?

---

@lbergelson 
I'll make the switch. why isn't sys.exit ideal?